check-setup.sh

```
@@ -0,0 +1,160 @@
+    fi
+
+    # Dizin izinleri
+    if [ -r "$APP_DIR" ] && [ -x "$APP_DIR" ]; then
+        log "✅ Dizin izinleri doğru"
+    else
+        log "⚠️  Dizin izinleri kontrol edilmeli"
+    fi
+}
+
+show_info(){
+    local IP=$(hostname -I | awk '{print $1}')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)